careers/technical-engineer.md

```diff
@@ -1,48 +1,78 @@
 
 ## Senior Engineer
 
-Success as a Senior Engineer is based on their ability to make the team and individuals on the team more effective. A Senior Engineer's technical impact should be focused on using technology to increase the likelihood that Packback succeeds as a business. A Senior Engineer at Packback should be focused on making a 10x impact on the team; which can only be done by empowering others, optimizing processes, and identifying the highest impact projects. They are proficient at project management.
+The senior engineer displays a mastery of the core competencies of software engineering. Strong technical knowledge and high productivity are table stakes at this level. Growth opportunities focus on developing impact, communication, and collaboration skills in preparation for the staff engineer role.
 
-**Technical Ability:** The Senior Engineer should be seen as a solid engineer who is a go-to expert in at least one domain (e.g. frontend, backend, data, databases, site-reliability-engineering). The Senior Engineer is capable of owning technical design for projects of moderate complexity (measured in multiple sprints), and understands the trade-offs in creating good software in their area. They hold a depth of knowledge in systems that enables them to debug effectively without issues. In addition to writing consistently high-quality (i.e. largely bug free, idiomatic, performant, secure, architecturally sound, appropriately abstracted) code they proactively stay abreast of industry best practices and trends, and have acquired at least one major skill outside of core coding such as monitoring, documentation, integration testing, visual design, performance optimization.
+**Technical Knowledge:** They have deep understanding of their technical specialty, and a working of knowledge of other domains. They take responsibility for the overall health and quality of their domain. They have responsibilities beyond writing code (managing audits, running on-call rotations, owning a backlog, etc.) to help them manage their domain within the organization. Feedback on their code usually revolves around subjective decisions, rather than objective issues. They proactively stay abreast of industry best practices and trends, and have acquired at least one major skill outside of core coding such as monitoring, documentation, integration testing, visual design, performance optimization.
```

Comment:
Minor addition to make the list clearer:

```suggestion
**Technical Knowledge:** They have deep understanding of their technical specialty, and a working of knowledge of other domains. They take responsibility for the overall health and quality of their domain. They have responsibilities beyond writing code (managing audits, running on-call rotations, owning a backlog, etc.) to help them manage their domain within the organization. Feedback on their code usually revolves around subjective decisions, rather than objective issues. They proactively stay abreast of industry best practices and trends, and have acquired at least one major skill outside of core coding such as monitoring, documentation, integration testing, visual design, and performance optimization.
```